src/routes/docs/tooling/command-line/installation/+page.markdoc

```diff
@@ -146,10 +146,10 @@ This will create your `appwrite.json` file, where you will configure your variou
 }
 ```
 
-You can run your first CLI command after logging in. Try fetching information about your Appwrite project.
+You can run your first CLI command after logging in. Try fetching the users from your Appwrite project.
 
 ```sh
-appwrite projects get --project-id "<PROJECT_ID>"
+appwrite users list
```

Comment:
```suggestion
appwrite users list --json
```

I would add --json here because the list is unreadable otherwise

![CleanShot 2024-08-24 at 17 32 37@2x](https://github.com/user-attachments/assets/52e46b6a-03ae-4629-aa7e-8539a72dbd67)